index.html

```diff
@@ -349,6 +349,23 @@ <h2>About Myself</h2>
 
 
 
+      </section>
+      <section class="contact-form">
+        <h2>Contact Me</h2>
+        <p>If you have an application you are interested in developing, a
+          feature that you need built or a project that needs coding.
+          I’d love to help with it!</p>
+        <form action="/contact" method="post" enctype="multipart/form-data">
+          <input type="text" placeholder="Full names"
+            max="25" min="10"
+            required="required">
+          <input type="text" placeholder="email" min="10"
+            required="required">
+          <textarea rows="7" placeholder="Write your message here"></textarea>
+          <input type="submit" value="Get in touch">
```

Comment:
- [ ] For good user experience, Please ensure that you implement button interactions such as hover, active (when it's pressed), disabled, and enable (default state) are styled as shown in the Figma template.

![image](https://github.com/NoelLincoln/mportfolio/assets/63473584/c705b2c2-eee2-46b4-bcb4-f19750469f76)